### DISCUSSION TITLE
Wiring multiple microbrite lights to groups with single transformer

### DISCUSSION THREAD

**Discussion by NWSAlpine** [tagyoureit/nodejs-poolController#704] [General]
I am getting all the stuff I need for my DIY controller setup. I have almost everything ordered but the last remaining part I am planning out is how to wire the pool lights. I have 2 microbrite lights in the main body and 2 microbrite lights in bubblers on the sun shelf. These 4 lights are currently wired together off a single transformer. 

In an ideal state I want to have 2 lights controlled as one group and the other 2 as a separate group. Is it recommended to have a dedicated transformer for each group (2 in my case) or have a single relay for the transformer and then each group on independent relays on the low voltage side? I thought I saw some discussions mention the software was smart enough to keep the transformer powered as long as one light group was active. 

It seems like having the transformer powered on all the time when at least one light is on and then switching the low voltage side to flip through the color modes would be preferred.

I have the sequent 8 relay card stacked on the Building Automation V4 8-Layer Stackable HAT.

**Comment by rstrouse** [comment]
You can accomplish this either way.  I don't know why, but in every instance I have seen, they always switch the high power side of the transformer.  I don't know it that helps with voltage spikes when switching or not.

**Comment by rock-crusher** [comment]
Since the lights are already powered by one transformer it is obviously of the adequate size for all 4 lights. No reason to buy another transformer just to separate each group.
I personally would go the way you mentioned:
**“a single relay for the transformer and then each group on independent relays on the low voltage side”**
And yes, Just as you said: 
**“the software **_IS_** smart enough to keep the transformer powered as long as one light group was active”**

**Comment by NWSAlpine** [comment]
Awesome. I will use the 3 relay approach with a single transformer.

**Comment by truvec** [comment]
I am wondering this as well. @NWSAlpine  did you set it up? Success?

I am installing 4 intellibrites would want individual control. Its always said "one transformer per switched light"

I looked into individual transformers (small and low cost) Ive seen a few but they don't seem robust.

Given the high cost of these lights I'm not into making poor choices.

Switching them with relays seems perfect reasonable.

Are we missing something?

**Reply by NWSAlpine** [reply]
@truvec I have not wired it yet. I built my entire enclosure and wired up my Pentair VSF pump, Aquarite, Hayward heatpro, and temp probes. 

Have some guests in town so I had to stop but it’s next on the list in a few days. After that all I have left is one intelliValve to install.

**Reply by truvec** [reply]
Well, I ended up with 6 Microbrites on the 8-Relay. The lights work but having issues with color setting and light circuit control.

@NWSAlpine How did you do?

**Reply by NWSAlpine** [reply]
@truvec I never ordered all my new lights as I was looking for a better Globrite alternative. I tried the microbrites but they are too dim. I am ordering the new Globrite with the updated body that is supposed to fix the cracking issue. Should be here next week and will rewire everything.

**Reply by truvec** [reply]
yeah, I'm pretty underwhelmed with my 6 microbrites.

**Comment by rock-crusher** [comment]
Hi there truvec,
Never heard of the "one transformer per switched light" but that’s not saying much…lol.
If you are using the Pentair 300 watt transformer for your lights you should be ok with all 4 intellibrites on the one transformer.
The reason I say this is I believe Pentair spec sheet lists those lights as 26 watts @ 12 VAC. 
The PS300 transformer is rated at 300 watts max on the output side, so I would think it would be more than capable of powering your 4 intellibrites at the same time with plenty of room to spare. 
truvec, I would hate to give you bad info here, so double check the combined wattage of all your intellibrite lights you plan on connecting to the transformer.

**Reply by truvec** [reply]
Not concerned from a capacity POV.

Just don't have a lot of data where these lights are relay switched on the 12vac side. 

When using pentair automation they only have HV relays so if you want two circuits you have 2 transformers.

I think it just might be that in world of pool automation there is HV (pumps and devices) and LV (Valves).

The other area of concern is will they void the warranty if wired this way? I have a 3 year warranty on them so id like to preserve that. 

but I think that concern can be handled by a quick rewire before having a service call..

**Reply by tagyoureit** [reply]
Rstrouse outlined this pretty well in the [wiki](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#lighting-circuits) and this matches with what the Intellibrite installation manual says.  You can have a single transformer that is powering your 4 lights and have 1-4 relays that switch individual or grouped lights between the transformers and the physical lights.

**Reply by truvec** [reply]
It's not documented by the wiki or the intellibrite manual. There is nothing between a transformer and the light(j box doesn't count).

Again, I also don't see any reason why this wouldn't work. I'll test soon and update here. (famous internet last words)

Thanks

**Reply by NWSAlpine** [reply]
@tagyoureit @rstrouse any guidance on how to configure this setup with the multi-relay board running REM and a full nixie controller? I have for now the LV transformer 120v switched on relay 3. On relay 5 I have the low voltage side switched. I am lost trying to set it up where I turn on the lights on the low voltage side relay 5 but also have it turn on relay 3. Everything I tried it also shuts off the transformer when switching color modes which I don't want. I tried light groups. I tried triggers in REM with delays. I can't figure it out.

**Reply by tagyoureit** [reply]
Why are you switching the 120v side?  I'd just leave it on.  The power draw will be miniscule when the LV lights are off.  If you really are concerned about it set the 120v to turn on when your pool turns on and turns off when you turn off your pool.  But don't include the 120v relay in any light groups.  njsPC doesn't know anything about the 120v vs LV so if you have them tied together it will try to set the 120v side just like the LV lights.

**Reply by NWSAlpine** [reply]
I was under the impression that the software was smart enough based on an earlier comment in the thread to know when a light was on and keep the transformer on. Even when switching light themes. After setting it up I realized it's not possible today. 

I wanted to switch the transformer on/off because it buzzes and transfers sound through the wall. I'm sharing one transformer for 2 light groups on different relays. 

The buzzing issue I will need to address anyways and figure out a new way to run the lights.

**Reply by Telemachus1977** [reply]
> I was under the impression that the software was smart enough based on an earlier comment in the thread to know when a light was on and keep the transformer on. Even when switching light themes. After setting it up I realized it's not possible today.
> 
> I wanted to switch the transformer on/off because it buzzes and transfers sound through the wall. I'm sharing one transformer for 2 light groups on different relays.
> 
> The buzzing issue I will need to address anyways and figure out a new way to run the lights.

I know this is a little bit of an old topic, but I am helping a friend set up the same scenario. 

Seems like the best solution is a circuit named "Activate Lights" that you turn on to activate the relay for the transformer. 

Then you will use the light groups just as they are programmed in the software. 

It is an extra step that you'll have to teach your family when they want to use the lights, but is certainly is worth the cost savings of not having multiple transformers. 

Alternatively, there custom script codes that can be employed, but I'm not quite sure which logic would work to keep the transformer on. Probably best just to use your human brain.
